doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,282 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 --------------------------
 
--
+- A new submodule for quasi-Monte Carlo, `scipy.stats.qmc`, was added
+- The documentation design was updated to use the same PyData-Sphinx theme as
+  other NumFOCUS packages like NumPy.
+- We now vendor and leverage the Boost C++ library to enable numerous
+  improvements for long-standing weaknesses in `scipy.stats`
```

Comment:
> if you'd like more highlights added let me know

It might be worth mentioning the other improvements to SciPy stats together, e.g.
```
`scipy.stats` has five new distributions, seven new (or overhauled) hypothesis test, a new function for bootstrapping, and many other enhancements.
```

Reply:
added, thanks